### PR TITLE
Update Shareable Viz docs with sections on creating access key and billing.

### LINKED ISSUE #1573: Documentations improvement for sharable URL: Setting up access keys for bucket in AWS
In current docs [configure-your-aws-s3-bucket-and-set-credentials](https://docs.kedro.org/en/latest/visualisation/share_kedro_viz.html#configure-your-aws-s3-bucket-and-set-credentials) we can add more information about how to create `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` after creating S3 bucket in AWS.

### PR DESCRIPTION
**Pull request by NeroOkwa**
Resolves #1573 

## Description

This PR was created to update the section of 'create access key' with more details and screenshots of the steps. 

Also added a new section on 'Billing'.  

## Checklist

- [x] Read the [contributing](/CONTRIBUTING.md) guidelines
- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Updated the documentation to reflect the code changes
- [ ] Added new entries to the `RELEASE.md` file
- [ ] Added tests to cover my changes

### CODE REVIEW

File: docs/source/share_kedro_viz.md [hunk 1]
```diff
@@ -39,6 +39,29 @@ pip install -r src/requirements.txt
 
 You can host your Kedro-Viz project on Amazon S3. You must first create an S3 bucket and then enable static website hosting. To do so, follow the [AWS tutorial](https://docs.aws.amazon.com/AmazonS3/latest/userguide/HostingWebsiteOnS3Setup.html) to configure a static website on Amazon S3.
 
+
+Once the s3 bucket is created, you'll need to creat an (Identity and Access Management) IAM user account, user group, and access keys:
```
Comment (tynandebold) [inline docs/source/share_kedro_viz.md:43]:
```suggestion
Once the S3 bucket is created, you'll need to create an Identity and Access Management (IAM) user account, user group, and generate the corresponding access keys. To do so:
```

File: docs/source/share_kedro_viz.md [hunk 2]
```diff
@@ -39,6 +39,29 @@ pip install -r src/requirements.txt
...
+
+Sign in to the [AWS Management Console](https://console.aws.amazon.com/s3/) and create an  IAM user account.
```
Comment (tynandebold) [inline docs/source/share_kedro_viz.md:45]:
```suggestion
Sign in to the [AWS Management Console](https://console.aws.amazon.com/s3/) and create an IAM user account.
```

File: docs/source/share_kedro_viz.md [hunk 3]
```diff
@@ -39,6 +39,29 @@ pip install -r src/requirements.txt
...
+For more information, see the official AWS documentation about [IAM Identities](https://docs.aws.amazon.com/IAM/latest/UserGuide/id.html).
+
+![](../meta/images/kedro_viz_share_credentials1.png)
+
+Create a user group from the IAM dashboard (ensuring the user group has full access to AWS S3 policy).
```
Comment (tynandebold) [inline docs/source/share_kedro_viz.md:50]:
```suggestion
Create a user group from the IAM dashboard, ensuring the user group has full access to the AWS S3 policy.
```

File: docs/source/share_kedro_viz.md [hunk 4]
```diff
@@ -39,6 +39,29 @@ pip install -r src/requirements.txt
...
+
+![](../meta/images/kedro_viz_share_credentials2.png)
+
+"A user group is a collection of IAM users. User groups simplify permissions management because you can grant, change, and remove permissions for multiple users at the same time". For more information, see the official AWS documentation about [IAM user groups](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_groups.html?icmpid=docs_iam_help_panel).
```
Comment (tynandebold) [inline docs/source/share_kedro_viz.md:54]: I don't think we need the copy/paste parts of the AWS documentation into ours.

```suggestion
For more information, see the official AWS documentation about [IAM user groups](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_groups.html?icmpid=docs_iam_help_panel).
```

File: docs/source/share_kedro_viz.md [hunk 5]
```diff
@@ -39,6 +39,29 @@ pip install -r src/requirements.txt
...
+
+Add the IAM user to the user group (this is only possible if the group has been created).
+
+![](../meta/images/kedro_viz_share_credentials3.png)
+
+Select the user, and select ‘create access key’. Follow the steps and create your access credential.
```
Comment (tynandebold) [inline docs/source/share_kedro_viz.md:60]:
```suggestion
Select the user, then select `Create access key`. Follow the steps and create your keys.
```

File: docs/source/share_kedro_viz.md [hunk 6]
```diff
@@ -69,3 +92,9 @@ Here's an example of the flow:
 All permissions and access control are controlled by AWS. It's up to you, the user, if you want to allow anyone to see your project or limit access to certain IP addresses, users, or groups.
 
 You can control who can view your visualisation using [bucket and user policies](https://docs.aws.amazon.com/AmazonS3/latest/userguide/using-iam-policies.html) or [access control lists](https://docs.aws.amazon.com/AmazonS3/latest/userguide/acls.html). See the official AWS documentation for more information.
+
+## Billing
+
+You pay for storing objects in your S3 buckets. This depends on your objects’ size, how long you stored the object during the month, and the storage class.
```
Comment (tynandebold) [inline docs/source/share_kedro_viz.md:98]:
```suggestion
You pay for storing objects in your S3 buckets. The amount you pay depends on your objects’ size, how long you stored the object during the month, and the storage class.
```

File: docs/source/share_kedro_viz.md [hunk 7]
```diff
@@ -69,3 +92,9 @@ Here's an example of the flow:
...
+
+See the official  [AWS documentation](https://aws.amazon.com/s3/pricing/?nc=sn&loc=4) for more information.
```
Comment (tynandebold) [inline docs/source/share_kedro_viz.md:100]:
```suggestion
See the official [AWS documentation](https://aws.amazon.com/s3/pricing/?nc=sn&loc=4) for more information. 
```

**Review by tynandebold — COMMENTED** [review]
Looking good. The RTD build [failed](https://readthedocs.org/projects/kedro-viz/builds/22492685/), so I couldn't preview it. Would be good to be able to see it on the page.

In lieu of that, I've left some text change comments. Overall, much more informative for our users. Thanks for this.

File: docs/source/share_kedro_viz.md (hunk 4 above)
Comment (NeroOkwa) [reply docs/source/share_kedro_viz.md:54]: Agreed.

**Comment by stichbury** [comment]
> Looking good. The RTD build [failed](https://readthedocs.org/projects/kedro-viz/builds/22492685/), so I couldn't preview it. Would be good to be able to see it on the page.
> 
The build is failing because the links to the image files are incorrect -- they contain an extra subfolder `meta` which doesn't exist here on the viz repo. It's a straightforward fix, I can do it now and the preview should appear soon.

I am tied up with my PR right now but will review this and merge it into my branch as soon as I can, so it'll get added to the docs when my changes do. Thanks @NeroOkwa @tynandebold!

**Pull request merged**